src/components/HTMLEngineProvider/HTMLRenderers/NextStepEmailRenderer.tsx

```diff
@@ -1,17 +1,12 @@
 import React from 'react';
+import type {CustomRendererProps, TPhrasing, TText} from 'react-native-render-html';
 import Text from '@components/Text';
 import useThemeStyles from '@hooks/useThemeStyles';
 
-type NextStepEmailRendererProps = {
-    tnode: {
-        data: string;
-    };
-};
-
-function NextStepEmailRenderer({tnode}: NextStepEmailRendererProps) {
+function NextStepEmailRenderer({tnode}: CustomRendererProps<TText | TPhrasing>) {
     const styles = useThemeStyles();
 
-    return <Text style={[styles.breakWord, styles.textLabelSupporting, styles.textStrong]}>{tnode.data}</Text>;
+    return <Text style={[styles.breakWord, styles.textLabelSupporting, styles.textStrong]}>{'data' in tnode && tnode.data}</Text>;
```

Comment:
Just please make sure it won't show `false` text here and it the places you made the same updates

Reply:
I have updated this to prevent any render of boolean values, please have a look